### ISSUE TITLE
2D Cell Segmentation Issue (Empty Output)

### ISSUE THREAD

**Issue by uoysip** [Project-MONAI/MONAI#936]
Hello,

I have made some modifications to the `spleen_segmentation_3d_lightning.ipynb` notebook to work on a 2D public dataset (from http://celltrackingchallenge.net/). However, the trained model only outputs black images. I believe the issue is with the loss function parameters or the transforms but I could not fix the issue after experimenting with different configurations. I have attached the notebook I am working on below:

https://gist.github.com/uoysip/624f94a27dc0da75be1970bf933bdf4a 

The image/labels are 2D 512 x 512 images (1, 512, 512). The labels are grayscale images with each cell having a different intensity, I have attached an example output below:

![](https://i.imgur.com/NhWGkUm.png)

From #835 I have tried plotting `val_outputs` without the `torch.argmax()` and I get an output that depicts the input image:

![](https://i.imgur.com/JcPzjui.png)

Any help would be greatly appreciated.

**Comment by wyli** [comment]
I think the labels with "each cell having a different intensity" make the segmentation task very difficult (and using DiceLoss is properly not ideal), an easier task would be using some transforms to convert the labels into cell vs non-cell binary maps, and train the networks to do the binary prediction
